hazelcast/src/main/java/com/hazelcast/scheduledexecutor/impl/ScheduledTaskStatisticsImpl.java

```diff
@@ -23,20 +23,24 @@
 import java.io.IOException;
 import java.util.concurrent.TimeUnit;
 
+import static com.hazelcast.internal.cluster.Versions.V3_10;
+
 public class ScheduledTaskStatisticsImpl
         implements ScheduledTaskStatistics, TaskLifecycleListener {
 
     private static final TimeUnit MEASUREMENT_UNIT = TimeUnit.NANOSECONDS;
 
-    private long runs;
+    private transient long createdAt;
```

Comment:
I would move `transient` fields below "regular" fields and group them, so it's easier to see which blocks are serialized and which are not.
```
    private static final TimeUnit MEASUREMENT_UNIT = TimeUnit.NANOSECONDS;

    private long runs;
    private long lastRunDuration;
    private long lastIdleDuration;
    private long totalRunDuration;
    private long totalIdleDuration;

    private transient long createdAt;
    private transient long firstRunStart;
    private transient long lastRunStart;
    private transient long lastRunEnd;
```